webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/RestExtractorService.scala

```diff
@@ -899,9 +899,12 @@ final case class RestExtractorService (
     for {
       description <- extractJsonString(json, "description")
       overridable <- extractJsonBoolean(json, "overridable")
```

Comment:
I think you can remove overridable here and in RestParameter as wel

Reply:
I'll make another pr for this, and prefere to merge it for more test on real Rudder!